build.gradle

```diff
@@ -764,42 +760,14 @@ project("spring-integration-bom") {
 	}
 }
 
-apply plugin: 'org.asciidoctor.convert'
-
-asciidoctor {
-	sourceDir = file("$docsDir")
-	sources {
-		include 'index.adoc'
-	}
-	backends = ['html5', 'docbook']
-	logDocuments = true
-	options = [
-		doctype: 'book',
-	]
-	attributes = [
-		docinfo: '',
-		toc2: '',
-		'compat-mode': '',
-		imagesdir: '',
-		stylesheet: "${rootProject.projectDir}/src/reference/asciidoc/stylesheets/golo.css",
-		'source-highlighter': 'highlightjs'
-	]
+task reference(type: GradleBuild) {
+  group = 'Documentation'
+	description = 'Generate the reference documentation'
+  buildFile = 'src/reference/build.gradle'
```

Comment:
Looks like you have removed too much from your changes. 
I guess that `'src/reference/build.gradle` still must be included to this change. Otherwise we just can't build the project at all.

Reply:
I wondered about that but figured you know more than I do. I have put those files back now. :)